pagopa/io-app
# chore(IT Wallet): [SIW-1159] Add XState issuance machine scaffolding

## Short description
This PR adds the initial scaffolding of the **IT Wallet activation** and **IT Wallet credential** issuance XState machines.

### State diagrams
- IT Wallet activation machine: https://stately.ai/registry/editor/embed/393bb84d-c50f-4184-a882-0fcf0fff8def?machineId=6392ceb2-a47e-4484-8723-af34a8765b65
- IT Wallet credential issuance machine: https://stately.ai/registry/editor/embed/393bb84d-c50f-4184-a882-0fcf0fff8def?machineId=f54b1d6b-9163-49a4-a6a5-2a1f560001ba

## List of changes proposed in this pull request
- Added `ItwEidIssuanceMachine` with actions and actors stub implementations
- Added `ItwEidIssuanceMachineContext` and `Provider`
- Added `ItwCredentialIssuanceMachine` with actions and actors stub implementations
- Added `ItwCredentialIssuanceMachineContext` and `Provider`
- Wrapped `ItwStackNavigator` with the `ItwEidIssuanceMachineProvider` and `ItwCredentialIssuanceMachineProvider`

## How to test
Static checks should pass.
From the I**T Wallet playground**, open the **Discovery** feature and check that machine events are correctly dispatched (navigation events).
